pwa-devdocs/src/venia-pwa-concept/setup/index.md

```diff
@@ -59,7 +59,7 @@ The currently recommended Magento version to use with PWA Studio is **2.3.1**, w
 **Example:**
 
 ```sh
-composer create-project --repository=https://repo.magento.com/ magento/project-community-edition:2.3.1 [destination directory]
+composer create-project --repository=https://repo.magento.com/ magento/project-community-edition:2.3.X [destination directory]
```

Comment:
There are a lot of specific references to `2.3.1` elsewhere in this document. Mind updating them all?

Reply:
Looping in @jcalcaben for his input

Reply:
A wild card instance probably wouldn't make sense for this because the current version of Venia does not work with Magento 2.3.0. When the next release of Venia occurs, it has to be compatible with the next version of Magento 2.3, but not necessarily with previous 2.3 versions.